docs/2_4_common_schema.adoc

```diff
@@ -855,1491 +807,1469 @@ Element `<fmiModelDescription><DefaultExperiment>` is defined as:
 
-All initial dimension sizes (i.e. prior to any configuration or reconfiguration) must be positive integers (i.e. not zero), so that no dimension is initially vanished.
+>|_(15)_
+|_<<continuous>> <<independent>>_
+|_All variables are a function of the continuous-time variable marked as <<independent>>._
+_Usually, this is `time`._
 
-_[This allows importing tools to ignore <<structuralParameter,`structural parameters`>> because that <<start>> value reflects the internal default setting of that <<structuralParameter,`structural parameter`>>._
-_The rationale for requiring positive start values for <<structuralParameter, `structural parameters`>> is that this avoids importers having to deal with vanishing dimensions if they do not want to deal with them (or even with changing sizes at all)._
-_If we allowed 0 dimension sizes for initial values, tools that do not even care about changing dimension sizes must be prepared to handle vanishing dimensions.]_
+>|_(16)_
+|_<<fixed>> <<structuralParameter>>_
+|_<<parameter,`Parameter`>> used in  `<Dimension>` element;  can be changed before initialization in <<ConfigurationMode>>._
 
-Changes to dimension sizes are constrained by the `min`/`max` attributes of the referenced <<structuralParameter,`structural parameters`>>, which can be any non-negative integer, including zero.
-Specifying a minimum size of zero on a <<structuralParameter,`structural parameter`>> allows any related dimension sizes to be changed to zero in *Configuration Mode* or *Reconfiguration Mode*, thus causing the respective array size to go to zero, which leaves the respective array variable without any active elements.
+>|_(17)_
+|_<<tunable>> <<structuralParameter>>_
+|_<<parameter,`Parameter`>> used in  `<Dimension>` element;  can be changed before initialization in <<ConfigurationMode>> and in <<ReconfigurationMode>>._
 
-The actual dimension sizes of arrays are also constrained by the FMU platform, due to memory and addressing constraints:
-Since the API functions to access variables and their values are constrained to `size_t` individual elements, platforms with addresses of less than 64-bit width will not be able to access elements beyond their addressing limits, neither will they be able to allocate enough memory or address space to represent such arrays.
-For these reasons implementations must take platform-specific constraints into account when changing dimension sizes, and must be prepared to handle the inability of the FMU to adjust to the desired sizes during *Configuration Mode* or *Reconfiguration Mode*.
+>|_(18)_
+|_<<clock>>_
+|_Variable that defines a <<clock>>._
 
-Changing any dimension of a variable in *Configuration Mode* or *Reconfiguration Mode* invalidates the variable's current value (including its <<start>> value).
-It should be noted that changing a <<structuralParameter,`structural parameter`>> might affect dimension sizes of several variables.
+>|_(19)_
```

Comment:
In the table there are two line for clocks. Delete line 19?

Reply:
I don´t understand: which file? which line 19?

Reply:
I now get it (after fixing (3) and (4) I found the reference and understood (19)): I dropped (19) and turned the (19) in the table into an (18). Restrictions to local variables apply still, no need to explain again for clocks.